clients/cobol-lsp-vscode-extension/src/extension.ts

```diff
@@ -91,5 +95,6 @@ export async function activate(context: vscode.ExtensionContext) {
 }
 
 export function deactivate() {
+    if (compileTaskProvider) { compileTaskProvider.dispose(); }
```

Comment:
Why this instead of `context.subscriptions.push(compileTaskProvider)` in activate?

Reply:
updated